### ISSUE TITLE
Break TR table if title of TD is too long?

### ISSUE THREAD

**Issue by sudwebdesign** [JiHong88/SunEditor#96]
Hi JiHong88,

When TD title is too long, html is breaked in source with v2.14.0

To reproduce :

Paste in source mode:
```
<table><tr><th colspan="5" align="left">error test : ok when title is shoten :</th></tr>
<tr><th colspan="5" align="left">Call Stack fake</th></tr>
<tr><th align="center">#</th><th align="left">Time</th><th align="left">Memory</th><th align="left">Function</th><th align="left">Location</th></tr>
<tr><td align="center">1</td><td align="center">0.0125</td><td align="right">231976</td><td>{main}(  )</td><td title="/PluXml-5.7/index.php">../index.php<strong>:</strong>0</td></tr>
<tr><td align="center">2</td><td align="center">0.0507</td><td align="right">366792</td><td>plxMotor::getInstance(  )</td><td title="/PluXml-5.7/index.php">../index.php<strong>:</strong>30</td></tr>
<tr><td align="center">3</td><td align="center">0.0507</td><td align="right">368680</td><td>plxMotor-&gt;__construct(  )</td><td title="/PluXml-5.7/core/lib/class.plx.motor.php">../class.plx.motor.php<strong>:</strong>53</td></tr>
<tr><td align="center">4</td><td align="center">0.0528</td><td align="right">380656</td><td>plxPlugins-&gt;loadPlugins(  )</td><td title="/PluXml-5.7/core/lib/class.plx.motor.php">../class.plx.motor.php<strong>:</strong>94</td></tr>
<tr><td align="center">5</td><td align="center">0.0533</td><td align="right">417808</td><td>plxPlugins::getInstance(  )</td><td title="/PluXml-5.7/core/lib/class.plx.plugins.php">../class.plx.plugins.php<strong>:</strong>86</td></tr>
</tbody></table>



<table><tr><th colspan="5" align="left">error test : nook when title is ??? :</th></tr>
<tr><th colspan="5" align="left">Call Stack fake</th></tr>
<tr><th align="center">#</th><th align="left">Time</th><th align="left">Memory</th><th align="left">Function</th><th align="left">Location</th></tr>
<tr><td align="center">1</td><td align="center">0.0125</td><td align="right">231976</td><td>{main}(  )</td><td title="/var/www/PluXml-5.7/index.php">../index.php<strong>:</strong>0</td></tr>
<tr><td align="center">2</td><td align="center">0.0507</td><td align="right">366792</td><td>plxMotor::getInstance(  )</td><td title="/var/www/PluXml-5.7/index.php">../index.php<strong>:</strong>30</td></tr>
<tr><td align="center">3</td><td align="center">0.0507</td><td align="right">368680</td><td>plxMotor-&gt;__construct(  )</td><td title="/var/www/PluXml-5.7/core/lib/class.plx.motor.php">../class.plx.motor.php<strong>:</strong>53</td></tr>
<tr><td align="center">4</td><td align="center">0.0528</td><td align="right">380656</td><td>plxPlugins-&gt;loadPlugins(  )</td><td title="/var/www/PluXml-5.7/core/lib/class.plx.motor.php">../class.plx.motor.php<strong>:</strong>94</td></tr>
<tr><td align="center">5</td><td align="center">0.0533</td><td align="right">417808</td><td>plxPlugins::getInstance(  )</td><td title="/var/www/PluXml-5.7/core/lib/class.plx.plugins.php">../class.plx.plugins.php<strong>:</strong>86</td></tr>
</tbody></table>
```

return to normal, the second table have lost TR
return to source mode and titles of 2nd table like this : 
`<td title="/em>`

<table>
<tbody>
<tr>
<th colspan="5" align="left">
error test : nook when title is ??? :</th>
</tr>
<tr>
<th colspan="5" align="left">
Call Stack fake</th>
</tr>
<tr>
<th align="center">
#</th>
<th align="left">
Time</th>
<th align="left">
Memory</th>
<th align="left">
Function</th>
<th align="left">
Location</th>
</tr>
<tr>
  <td align="center">1</td>
  <td align="center">0.0125</td>
  <td align="right">231976</td>
  <td>{main}(  )</td>
  <td title="/em>../index.php<strong>:</strong>0</td>
</tr>
<tr>
  <td align=" center"="">2</td>
  <td align="center">0.0507</td>
  <td align="right">366792</td>
  <td>plxMotor::getInstance(  )</td>
  <td title="/em>../index.php<strong>:</strong>30</td>
</tr>
<tr>
  <td align=" center"="">3</td>
  <td align="center">0.0507</td>
  <td align="right">368680</td>
  <td>plxMotor-&gt;__construct(  )</td>
  <td title="/em>../class.plx.motor.php<strong>:</strong>53</td>
</tr>
<tr>
  <td align=" center"="">4</td>
  <td align="center">0.0528</td>
  <td align="right">380656</td>
  <td>plxPlugins-&gt;loadPlugins(  )</td>
  <td title="/em>../class.plx.motor.php<strong>:</strong>94</td>
</tr>
<tr>
  <td align=" center"="">5</td>
  <td align="center">0.0533</td>
  <td align="right">417808</td>
  <td>plxPlugins::getInstance(  )</td>
</tr>
</tbody>
</table>

FULL SOURCE CODE:
```
<table>
<tbody>
<tr>
<th colspan="5" align="left">
error test : ok when title is shoten :</th>
</tr>
<tr>
<th colspan="5" align="left">
Call Stack fake</th>
</tr>
<tr>
<th align="center">
#</th>
<th align="left">
Time</th>
<th align="left">
Memory</th>
<th align="left">
Function</th>
<th align="left">
Location</th>
</tr>
<tr>
  <td align="center">1</td>
  <td align="center">0.0125</td>
  <td align="right">231976</td>
  <td>{main}(  )</td>
  <td title="/PluXml-5.7/index.php">../index.php<strong>:</strong>0</td>
</tr>
<tr>
  <td align="center">2</td>
  <td align="center">0.0507</td>
  <td align="right">366792</td>
  <td>plxMotor::getInstance(  )</td>
  <td title="/PluXml-5.7/index.php">../index.php<strong>:</strong>30</td>
</tr>
<tr>
  <td align="center">3</td>
  <td align="center">0.0507</td>
  <td align="right">368680</td>
  <td>plxMotor-&gt;__construct(  )</td>
  <td title="/PluXml-5.7/core/lib/class.plx.motor.php">../class.plx.motor.php<strong>:</strong>53</td>
</tr>
<tr>
  <td align="center">4</td>
  <td align="center">0.0528</td>
  <td align="right">380656</td>
  <td>plxPlugins-&gt;loadPlugins(  )</td>
  <td title="/PluXml-5.7/core/lib/class.plx.motor.php">../class.plx.motor.php<strong>:</strong>94</td>
</tr>
<tr>
  <td align="center">5</td>
  <td align="center">0.0533</td>
  <td align="right">417808</td>
  <td>plxPlugins::getInstance(  )</td>
  <td title="/PluXml-5.7/core/lib/class.plx.plugins.php">../class.plx.plugins.php<strong>:</strong>86</td>
</tr>
</tbody>
</table>
<table>
<tbody>
<tr>
<th colspan="5" align="left">
error test : nook when title is ??? :</th>
</tr>
<tr>
<th colspan="5" align="left">
Call Stack fake</th>
</tr>
<tr>
<th align="center">
#</th>
<th align="left">
Time</th>
<th align="left">
Memory</th>
<th align="left">
Function</th>
<th align="left">
Location</th>
</tr>
<tr>
  <td align="center">1</td>
  <td align="center">0.0125</td>
  <td align="right">231976</td>
  <td>{main}(  )</td>
  <td title="/em>../index.php<strong>:</strong>0</td>
</tr>
<tr>
  <td align=" center"="">2</td>
  <td align="center">0.0507</td>
  <td align="right">366792</td>
  <td>plxMotor::getInstance(  )</td>
  <td title="/em>../index.php<strong>:</strong>30</td>
</tr>
<tr>
  <td align=" center"="">3</td>
  <td align="center">0.0507</td>
  <td align="right">368680</td>
  <td>plxMotor-&gt;__construct(  )</td>
  <td title="/em>../class.plx.motor.php<strong>:</strong>53</td>
</tr>
<tr>
  <td align=" center"="">4</td>
  <td align="center">0.0528</td>
  <td align="right">380656</td>
  <td>plxPlugins-&gt;loadPlugins(  )</td>
  <td title="/em>../class.plx.motor.php<strong>:</strong>94</td>
</tr>
<tr>
  <td align=" center"="">5</td>
  <td align="center">0.0533</td>
  <td align="right">417808</td>
  <td>plxPlugins::getInstance(  )</td>
</tr>
</tbody>
</table>
```

**Comment by sudwebdesign** [comment]
Tested with Firefox 66 and with or without option

**Comment by JiHong88** [comment]
Hi @sudwebdesign 

Fixed in [v2.15.0](https://github.com/JiHong88/SunEditor/releases/tag/2.15.0).

Thank you :)
